docs/int/quickstart/quickstart-alone.md

```diff
@@ -25,34 +27,93 @@ Charon is in an early alpha state and is not ready to be run on mainnet
    # Copy the sample environment variables
    cp .env.sample .env
    ```
-
    `.env.sample` is a sample environment file that allows overriding default configuration defined in `docker-compose.yml`. Uncomment and set any variable to override its value.
 
 1. Create the artifacts needed to run a testnet distributed validator cluster
 
    ```sh
    # Create a testnet distributed validator cluster
-   docker run --rm -v "$(pwd):/opt/charon" obolnetwork/charon:v0.11.0 create cluster --withdrawal-address="0x000000000000000000000000000000000000dead"
+   docker run --rm -v "$(pwd):/opt/charon" ghcr.io/obolnetwork/charon:v0.11.0 create cluster --withdrawal-address="0x000000000000000000000000000000000000dead" --nodes 6 --threshold 5
```

Comment:
This shouldn't use the `gchr.io` container repo, that will force people to log into github on that box to pull the image. Seems like this was copied across from stale docs probably in the readme I'm guessing.

Reply:
removed in next PR https://github.com/ObolNetwork/obol-docs/pull/77